docs/content/documentation/systems/AuxKernels/framework/SolutionAux.md

```diff
@@ -1,6 +1,7 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # SolutionAux
+
+This AuxKernel works in conjunction with a [SolutionUserObject](/SolutionUserObject.md) to produce fields. If the solution was created on an equivalent mesh, the direct option can be used to read using DOF information. Otherwise, a point locator is used to find the nearest node or element and the corresponding value. Finally, these values can be scaled and/or added by utilizing the *scale_factor* or *add_factor* parameters.
```

Comment:
We should define a standard for writing parameters, we use `param`, "param", 'param', and *param*. We should pick one an try to stick with it.